src/components/UpcomingQuizCard/icon/CalendarIcon.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, type SxProps, type Theme } from '@mui/material';
+import React from 'react';
+
+export interface CalendarIconProps {
+  day?: number | string;
+  fontSize?: 'small' | 'medium' | 'large';
+  color?: string;
+  sx?: SxProps<Theme>;
+}
+
+const CalendarIcon: React.FC<CalendarIconProps> = ({
+  day = new Date().getDate(),
+  fontSize = 'medium',
+  color = '#1976d2',
+  sx,
+}) => {
+  const sizeMap = {
+    small: 32,
+    medium: 48,
+    large: 64,
+  };
+
+  const size = sizeMap[fontSize] || 48;
+
+  return (
+    <Box component="span" sx={{ display: 'inline-flex', alignItems: 'center', ...sx }}>
+      <svg width={size} height={size} viewBox="0 0 64 64" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVG element does not provide an accessible name or role. If the icon conveys meaningful information (the day), it should be accessible (role and aria-label or visually-hidden text).

**Sugerencia:**
```typescript
Add role and accessible name when appropriate: `<svg role="img" aria-label={`Calendar day ${day}`} ...>` or allow passing an `aria-label` prop. Alternatively, if the calendar icon is purely decorative, add `aria-hidden="true"`.
```